src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -1097,11 +1099,8 @@ export class ServerView extends DirMixin(PolymerElement) {
   }
 
   // Takes language so that the server location is recalculated on app language change.
-  _formatLocation(serverLocationId, UNUSED_language) {
-    if (!serverLocationId) {
-      return '';
-    }
-    return this.localize(`city-${serverLocationId}`);
+  _formatLocation(serverLocation, UNUSED_language) {
```

Comment:
Pass the localize function as a parameter instead of the language, so it gets properly updated on language change.
You won't need the language if we're not localizing the country.

Reply:
As we discussed, I believe we can address this by creating a UI Location that has a country code but no location_msg. Whoever creates the UI Locations can check if the location is a country and fill the object accordingly.

Reply:
I went with "if the GeoId and the CountryCode are equal, omit the country".